docs/en/serving-tiles/manually-building-a-tile-server-debian-13.md

```diff
@@ -0,0 +1,414 @@
+sudo -u _renderd psql -d gis -f functions.sql
+```
+
+### Shapefile download
+
+Although most of the data used to create the map is directly from the OpenStreetMap data file that you downloaded above, some shapefiles for things like low-zoom country boundaries are still needed. To download and index these, using the same account as we used previously:
+
+```sh
+cd ~/src/openstreetmap-carto/
+mkdir data
+sudo chown _renderd data
+sudo -u _renderd scripts/get-external-data.py
+```
+
+This process involves a sizable download and may take some time - not much will appear on the screen when it is running. Some data will go directly into the database, and some will go into a “data” directory below “openstreetmap-carto”. If there is a problem here, then the Natural Earth data may have moved - look at [this issue](https://github.com/nvkelso/natural-earth-vector/issues/581#issuecomment-913988101){: target=_blank} and similar ones at Natural Earth for more details. If you need to change the Natural Earth download location your copy of [this file](https://github.com/gravitystorm/openstreetmap-carto/blob/master/external-data.yml){: target=_blank} is the one to edit.
+
+### Fonts
+
+Fonts need to be installed manually, like this:
+
+```sh
+cd ~/src/openstreetmap-carto/
+scripts/get-fonts.sh
+```
+
+Due to a [current issue](https://github.com/gravitystorm/openstreetmap-carto/issues/5013) you might get an error at the very end of that, but it won't adversely affect the display of any current OSM data.
+
+Our test data area (Azerbaijan) was chosen both because it was a small area and because some place names in that region have names containing non-latin characters.
+
+## Setting up your webserver
+
+### Configure renderd
+
+The config file for `renderd` on {{ dist }} is `/etc/renderd.conf`. Edit that with a text editor such as nano:
+
+```sh
+sudo nano /etc/renderd.conf
+```
+
+Add a section like the following at the end:
+
+```ini
+[s2o]
+URI=/hot/
+TILEDIR=/var/lib/mod_tile
+XML=/home/accountname/src/openstreetmap-carto/mapnik.xml
+HOST=localhost
+TILESIZE=256
+MAXZOOM=20
+```
+
+The location of the XML file `/home/accountname/src/openstreetmap-carto/mapnik.xml` will need to be changed to the actual location on your system. You can change `[s2o]` and `URI=/hot/` as well if you like. If you want to render more than one set of tiles from one server you can - just add another section like `[s2o]` with a different name referring to a different map style. If you want it to refer to a different database to the default `gis` you can, but that's out of the scope of this document. If you've only got 4Gb or so of memory, you'll also want to reduce `num_threads` to 2. `URI=/hot/` was chosen so that the tiles generated here can more easily be used in place of the HOT tile layer at OpenStreetMap.org. You can use something else here, but `/hot/` is as good as anything.
+
+The location of the Mapnik plugins directory now varies archtecture.  You can type `` 
```

Comment:
```suggestion
The location of the Mapnik plugins directory now varies architecture.  You can type
```

Reply:
Thanks - I've fixed that (reworked the sentence to be better English), and also mentioned that you need to run "updatedb" before running locate.